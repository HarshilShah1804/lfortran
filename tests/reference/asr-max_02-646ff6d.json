{
    "basename": "asr-max_02-646ff6d",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/max_02.f90",
    "infile_hash": "c2180a170462597a12e62ea4c43eb4c64bd2a4759f66c8d393a12326",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-max_02-646ff6d.stdout",
    "stdout_hash": "c198e6cd162e623ee7e666b83aef8923c47e61d5a43f9d6c714cf255",
    "stderr": "asr-max_02-646ff6d.stderr",
    "stderr_hash": "97e360a47fa5edf6fba380dbc25904287f1994bc0091a7a77d421c5e",
    "returncode": 0
}