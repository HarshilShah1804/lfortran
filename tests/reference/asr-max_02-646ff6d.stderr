warning: Different kinds of args in max0 is a non-standard extension
 --> tests/../integration_tests/max_02.f90:4:14
  |
4 |     print *, max(y, z)
  |              ^^^^^^^^^ help: ensure all arguments have the same kind to make it standard

warning: Different kinds of args in max0 is a non-standard extension
 --> tests/../integration_tests/max_02.f90:5:41
  |
5 |     print *, "Kind of max(y, z):", kind(max(y,z))
  |                                         ^^^^^^^^ help: ensure all arguments have the same kind to make it standard

warning: Different kinds of args in max0 is a non-standard extension
 --> tests/../integration_tests/max_02.f90:6:14
  |
6 |     if (kind(max(y,z)) /= 8) error stop "Incorrect kind of max"
  |              ^^^^^^^^ help: ensure all arguments have the same kind to make it standard
